### PR TITLE
Activate github workflows

### PR DESCRIPTION
**Pull request by jan-kiszka**
They lifted the time limit per job to 6 hours which even permits running the example image build followed by the RT image in a row (reusing build artifacts). This also enables keeping the artifacts (up to 90 days) for direct testing purposes.

In addition, this comes with an Isar update, getting us down to zero patches again.

### CODE REVIEW

**Comment by BaochengSu** [comment]
Great work!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  images:
+    name: Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Example image
+        run: ./kas-docker build kas-iot2050-example.yml
```
Comment (BaochengSu) [inline .github/workflows/main.yml:13]: Just curious, why not use kas-container directly?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Upload example image
+        uses: actions/upload-artifact@v2
+        with:
+          name: iot2050-example-image
+          path: build/tmp/deploy/images/iot2050/iot2050-image-example-isar-iot2050.wic.img*
```
Comment (BaochengSu) [inline .github/workflows/main.yml:18]: How many disk spaces do we have?

File: .github/workflows/main.yml (hunk 1 above)
Comment (jan-kiszka) [reply .github/workflows/main.yml:13]: Oops, legacy

File: .github/workflows/main.yml (hunk 2 above)
Comment (jan-kiszka) [reply .github/workflows/main.yml:18]: No idea. I didn't find notes on that on github yet. Maybe you can try to search as well. But if there is a limit: There is also no hint how much you used already.

Comment (BaochengSu) [reply .github/workflows/main.yml:18]: > No idea. I didn't find notes on that on github yet. Maybe you can try to search as well. But if there is a limit: There is also no hint how much you used already.

I tried to search it but didn't find anything, I am wondering if we are using the organization storage space...

**Pull request merged**